docker-compose.yml

```diff
@@ -0,0 +1,29 @@
+services:
+  db:
+    image: mysql:9.2.0
+    environment:
+      - MYSQL_DATABASE=orte
+      - MYSQL_USER=orte
+      - MYSQL_PASSWORD=orte00
+      - MYSQL_ALLOW_EMPTY_PASSWORD=true
+    healthcheck:
+      test: [ "CMD", "mysqladmin", "ping", "-h", "localhost" ]
+      interval: 10s
+      timeout: 5s
+      retries: 5
+  server:
+    build:
+      context: .
+      dockerfile: Dockerfile
+      args:
+        - RAILS_MASTER_KEY=${RAILS_MASTER_KEY}
```

Comment:
Not required if this were on Rails 7.1: https://hayford.dev/this-week-in-rails-wrapped-an-overview-of-rails-7-1-features-part-iii/